src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```
@@ -50,13 +50,12 @@ tests:
               minValue: 0
               maxValue: 32768
 
-    #Issue 18041
     - label: "TH reads from the DUT the (0xFFFB) AttributeList attribute"
       PICS: PICS_SKIP_SAMPLE_APP
```

```suggestion
      PICS: PICS_SKIP_SAMPLE_APP
```
are we sure about this pics ?

---

Hi @jmeg-sfy @raju-apple ya we added this PICS just to skip steps that are currently not supported or do not run on the CI, If this [PR](https://github.com/project-chip/connectedhomeip/pull/19730) merges we will remove this PICS condition

---

ok good to know thanks